mfc.sh

```diff
@@ -159,10 +159,26 @@ elif [ "$1" == "format" ]; then
         pip3 install --upgrade fprettify
     fi
 
+    if [ "$1" == "diff" ]; then 
+        shift
+        out=$(fprettify ${@:-src} --exclude "src/*/autogen" --recursive --silent \
+            --indent 4 --c-relations --enable-replacements --enable-decl \
+            --whitespace-comma 0 --whitespace-multdiv 0 --whitespace-plusminus 1 \
+            --case 1 1 1 1 --strict-indent --line-length 1000 -s -d)
+        if [ -z "${out}" ]; then
```

Comment:
I usually prefer counting the lines but this just as correct as far as I can tell.

Reply:
Yeah, I checked it.